graphql/client_test.go

```diff
@@ -0,0 +1,83 @@
+package graphql
+
+import (
+	"github.com/pinpt/go-common/hash"
+	"github.com/stretchr/testify/assert"
+	"math/rand"
+	"os"
+	"testing"
+)
+
+func TestClient(t *testing.T) {
+	if os.Getenv("CI") != "" {
```

Comment:
we should also write unit tests that use httptest and mock out graphql endpoint

Reply:
let's discuss this offline

Reply:
I wasn't familiar with the `httptest` package, I will modify the unit tests in a bit. Thanks for looking out